Registry/Registry.EM_COMMON

```diff
@@ -2245,6 +2252,7 @@ rconfig   integer shcu_aerosols_opt       namelist,physics      max_domains    0
 
 rconfig   integer ICLOUD_CU               derived               max_domains   0        -     "ICLOUD_CU"                     ""      ""
 rconfig   integer pxlsm_smois_init        namelist,physics      max_domains   1       irh    "PXLSM_SMOIS_INIT"    "Soil moisture initialization option 0-From analysis 1-From MAVAIL"   ""
+rconfig   integer pxlsm_modis_veg         namelist,physics      max_domains   0       irh    "PXLSM_MODIS_VEG"     "If 1 use MODIS VEGFRA from wrflowinp updates, 1=yes"   ""
```

Comment:
@coastwx 
Have you tested a nested scenario where one domain used the data from the wrflowinp file and another domain did not? If this is not a recommended scenario, a test in check_a_mundo should be added. You could either make all of the domains the same as d01 (silently), or you could issue a fatal error.